python/cudf/cudf/io/orc.py

```diff
@@ -421,6 +421,14 @@ def to_orc(
             "Categorical columns."
         )
 
+    # Convert from ORC terminology ("STRIPE", "ROWGROUP")
+    # to CUDF ("ROWGROUP", "PAGE")
+    stats_freq = "NONE"
+    if enable_statistics == "STRIPE":
+        stats_freq = "ROWGROUP"
+    elif enable_statistics == "ROWGROUP":
+        stats_freq = "PAGE"
+
```

Comment:
The statistics tests are failing because we are trying to do the conversion from ORC to Parquet naming twice - once here and again in `_get_orc_stat_freq`. AFAICT, the conversion here is not required and tests are passing locally for me when this section is removed.
```suggestion
```

Reply:
That's odd. That was failing for me locally. I'll try removing that section.

Reply:
I'm not sure what I missed yesterday. I have verified that your suggestion works.